.github/workflows/update_translations_from_source.yml

```diff
@@ -0,0 +1,34 @@
+name: Update Translations from Source
+
+on:
+  push:
+    branches:
+      - master
+      - auto-update-translations
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron:  '0 0 * * *'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Setup GitHub Checkout
+      uses: actions/checkout@v2
+    - name: Setup PHP Action
+      uses: shivammathur/setup-php@1.8.7
+      with:
+        php-version: '7.4'
+    - name:  Update Translations from Source
+      run: |
+        php _backend/Console/Translations.php
+        echo "Setting up git credentials..."
```

Comment:
I'd probably split this to two steps. One to run the PHP script, and another to push the files.